### ISSUE TITLE
sdss_install breaks with new setup_tree script

### ISSUE THREAD

**Issue by havok2063** [sdss/sdss_install#9]
`sdss_install` does not work with the recent changes to the tree installation.  The tree installation has been simplified to a single python script `bin/setup_tree.py`.  

Traceback:
```
$ sdss_install tree master -G -F
/uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/sdss_install/master/python/sdss_install/init.py:31: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
config = yaml.load(open(os.path.dirname(file) + '/etc/{0}.yml'.format(NAME)))
sdssinstall - INFO - Preparing to install in /uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/tree/master (overwriting due to force option)
sdssinstall - INFO - Running command: git clone https://github.com/sdss/tree.git tree-master
sdssinstall - INFO - Completed GitHub clone of repository tree
sdssinstall - INFO - Installing in /uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/tree/master
sdssinstall - INFO - Running "make" in /uufs/chpc.utah.edu/common/home/sdss10/software/sdss/github/tree/master
sdssinstall - ERROR - Error during compile:
sdssinstall - ERROR - make[1]: ../bin/sas_base.sh: Command not found
make[1]: ../bin/install_tree.py: Command not found
make[1]: *** [all] Error 127
make: *** [all] Error 2

sdssinstall - INFO - Fail!
```

**Comment by havok2063** [comment]
I've pushed a fix to `tree/master` where this should now be fixed.  Since other packages may use `Makefiles`, I kept the `Makefile` inside `tree/etc/Makefile` and updated it to point to `bin/setup_tree.py`.  `sdss_install -G -F tree master` now seems to work for me.  

New environment files are created in the `etc` directory and module files are copied over to the user's specified modules directory.  If no module_files path is specified in `setup_tree.py`, it will default to looping over every path specified in `$MODULEPATH`, and it will copy the module files into those directories have a `tree` subfolder.  This mimics the original `install_tree.sh` script.  

This assumes that when a user runs `sdss_install` the modulefiles path that it uses has been added into the `$MODULEPATH` somehow, e.g. using `module use`.  This is currently a manual step that has to be done before the `sdss_install_boostrap` step, however I think this should be explicitly set by `sdss_install` at least when installing the `tree`.
